tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -221,6 +282,33 @@ def do_log_copy(args: CensusBuildArgs) -> bool:
     return True
 
 
+def do_sync_release_file_to_replica_s3_bucket(args: CensusBuildArgs) -> bool:
```

Comment:
These new operations (as well as do_the_release) need to have checks that the build was done.

For example, `do_the_release` should contain an assertion like:

> assert args.state['do_validate_soma'] is True
> assert args.state['do_data_copy'] is true

I.e., to confirm that steps it depends on were succesful.

Honestly, I'd like to see _all_ do_foo functions add these assertions (now that we allow potential for out-of-order accidental execution)

Reply:
Discussed offline. We'll introduce a shared state on S3 in the next iteration.